components/molecules/Carousel/index.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Carousel = () => (
+  <div></div>
```

Comment:
maybe better use fragments? https://reactjs.org/docs/fragments.html

Reply:
> maybe better use fragments? https://reactjs.org/docs/fragments.html

ok. We will make this changes in the next step